src/components/TeamMembers/TeamMemberDialogDisplay.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { Dialog, Typography, CardActions } from '@material-ui/core';
+import teamMemberStyles from "./TeamMemberStyle";
+import { LinkedIn, Mail, GitHub, LocalPhone, Info } from '@material-ui/icons';
+
+const DialogDisplay = ({ person: { name, designation, phoneNumber, emailId, githubProfile, linkedinProfile } }) => {
+    const classes = teamMemberStyles();
+    const [open, setOpen] = useState(false);
+    return (
+        <div className={`${classes.tilebar} ${classes.tilebar} ${classes["tilebar-root-title"]} ${classes["tilebar-bottom"]}`}>
+            <div className={`${classes["title-pos-right"]} ${classes["title-wrap"]}`}>
+                <div className={classes.title}>{name}</div>
+                <div><span>{designation}</span></div>
+            </div>
+            <CardActions onClick={() => setOpen(true)} className={classes["action-item"]}><Info /></CardActions>
+            <Dialog
+                aria-labelledby="simple-dialog-title"
+                aria-describedby="simple-dialog-description"
+                open={open}
+                onClose={() => { setOpen(false) }}
+                BackdropProps={{
+                    classes: {
+                        root: classes.backdrop
+                    }
+                }}
+                PaperProps={{
+                    style: {
+                        boxShadow: 'none'
```

Comment:
Let it be. A modal should have a background.

Reply:
The box shadow was looking odd. So, i removed it.